scripts/run_translate_schemas.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+current_dir=$(dirname "${BASH_SOURCE[0]}")
+parent_dir=$(dirname "${current_dir}")
+path_to_parent="$( cd "${parent_dir}" && pwd )"
+
+# Clone translation schema
+git clone https://github.com/ONSdigital/eq-translations.git "${path_to_parent}"/temp/eq-translations
```

Comment:
Actually because all of the other stuff is done in a sub shell it is okay. If it fails it will still run the `rm -rf`